test/execution_plan_test.rb

```diff
@@ -191,6 +191,18 @@ module ExecutionPlanTest
         end
       end
 
+      describe 'custom plan id' do
+        let :execution_plan do
+          world.plan_with_options(action_class: Support::CodeWorkflowExample::IncomingIssues,
+                                  args: [issues_data],
+                                  id: 'my-unique-id')
```

Comment:
If I recall on postgres we use an `UUID` typed column for storing the id. Passing something what is not an uuid should fail. Any idea why that doesn't happen?

Reply:
Updated to use uuid. It turns out we were not using uuid for postgres due to incorrect database condition. See https://github.com/Dynflow/dynflow/pull/319 for more details.